angular/src/lib/select/select.component.ts

```diff
@@ -611,12 +613,18 @@ export class SelectComponent implements AfterContentChecked, AfterContentInit, C
     if (this.filterViewChild && this.filterViewChild.nativeElement) {
       this.filterViewChild.nativeElement.focus();
     }
+    const activeItem = document.querySelector('.md-select-item--focus');
```

Comment:
Do we need to support the browser which does not support querySelector?

Reply:
Looks to be supported by all browsers appearing here; https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector